src/TypeScript/EslintBridgeClient/EslintBridgeStartUp.cs

```diff
@@ -0,0 +1,35 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Threading.Tasks;
+
+namespace SonarLint.VisualStudio.TypeScript.EslintBridgeClient
+{
+    // TODO: https://github.com/SonarSource/sonarlint-visualstudio/issues/2256
+
+    internal interface IEslintBridgeStartUp : IDisposable
+    {
+        /// <summary>
+        /// Ensures that the process is running. Returns the port that server is running on.
+        /// </summary>
+        Task<int> Start();
```

Comment:
Not so sure yet about this method and the "close" functionality. Created a dedicated ticket #2256